miso-web/src/main/java/uk/ac/bbsrc/tgac/miso/webapp/controller/rest/PoolRestController.java

```diff
@@ -280,4 +280,12 @@ private PoolPickerEntry poolTransform(Pool pool) throws IOException {
         .map(completion -> Dtos.asDto(completion)).collect(Collectors.toList());
     return new PoolPickerEntry(Dtos.asDto(pool, true), completions);
   }
+
+  @RequestMapping(value = "query", method = RequestMethod.POST, produces = { "application/json" })
+  @ResponseBody
+  public List<PoolDto> getSamplesInBulk(@RequestBody List<String> names, HttpServletRequest request, HttpServletResponse response,
+      UriComponentsBuilder uriBuilder) throws IOException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'java.io.IOException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)